modules/gateways/blockonomics/payment.tpl

```diff
@@ -0,0 +1,144 @@
+<link rel="stylesheet" type="text/css" href="/modules/gateways/blockonomics/css/order.css">
```

Comment:
Links/scripts on this page should rather use relative paths eg. `../blockonomics/css/order.css` This is for users who install whmcs in the non-root directory of their server. eg. https://website.com/whmcs.

Reply:
Even better: `{$WEB_ROOT}/modules/gateways/blockonomics/css/order.css ` as used in templates/orderforms

Reply:
Yes {$WEB_ROOT} looks like a good solution.